app/views/wizards/new_objective.html.slim

```diff
@@ -0,0 +1,24 @@
+.columns.is-centered
+  .column.is-one-third
+    h2.title.is-4.has-text-centered Step 3
```

Comment:
<img width="814" alt="image" src="https://github.com/widefix/pocketmoney/assets/243846/908da40c-3970-4b06-979a-bbabc1441e4d">

Reply:
Note, "price" but not amount. No need to change in DB for now.